### PR TITLE
Added POQ.gg

### PR DESCRIPTION
**Pull request by Mystik01**
- Added Homepage
- Account Settings, can be enabled/disabled in settings.
- Added prize store 
- Added accounts
- Added buttons to view tournaments
- Added ToS page

image
image
image
image

image
image
image
image

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,5 +38,6 @@
     "source-map-support": "^0.5.19",
     "terser": "^5.6.0",
     "typescript": "^3.9.7"
-  }
+  },
+  "dependencies": {}
```
Comment (Bas950) [inline package.json:42]: undo

File: websites/P/POQ/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  },
+  "url": ["www.poq.gg", "prizes.poq.gg", "creators.poq.gg", "account.poq.gg"],
+  "description": {
+    "en": "POQ is a platform designed for creators to host and share tournaments! Players can join tournaments and be rewarded in Quarters which they can then spend in the prize store on gaming accessories.",
+    "es": "¡POQ es una plataforma diseñada para que los creadores organicen y compartan torneos! Los jugadores pueden unirse a torneos y ser recompensados en cuartos, que luego pueden gastar en la tienda de premios en accesorios de juego."
+  },
+  "service": "POQ",
+  "version": "1.0.0",
+  "logo": "https://pbs.twimg.com/profile_images/1303313694546423808/PAaYFaSK_400x400.jpg",
+  "thumbnail": "https://i.gyazo.com/7a15adb09474fd4103ac1a64fa5590cb.jpg",
```
Comment (Bas950) [inline websites/P/POQ/dist/metadata.json:15]: upload to imgur

File: websites/P/POQ/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  "color": "#FDE21F",
+  "tags": ["gaming", "fortnite", "tournaments", "poq", "esports"],
+  "category": "games",
+  "settings": [
+    {
+      "id": "prize",
+      "title": "Show what item you're viewing in the prize store.",
```
Comment (Bas950) [inline websites/P/POQ/dist/metadata.json:22]: make the title shorter

File: websites/P/POQ/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      "icon": "fas fa-shopping-cart",
+      "value": true
+    },
+    { 
+      "id": "pass",
+      "title": "Show when you're editing you account info",
```
Comment (Bas950) [inline websites/P/POQ/dist/metadata.json:28]: same here

File: websites/P/POQ/presence.ts [hunk 5]
```diff
@@ -0,0 +1,227 @@
+const presence = new Presence({
+  clientId: '752138871709499462'
+}),
+  browsingStamp = Math.floor(Date.now() / 1000);
+function cal(str: string) {
+  // -------------------------------------------------------- SEE LINE 70 ------------------------------------------------------------
+  // converting first letter to uppercase
```
Comment (Bas950) [inline websites/P/POQ/presence.ts:7]: no need for the comment

**Review by Bas950 — CHANGES_REQUESTED** [review]
Maybe use `else if`

File: package.json (hunk 1 above)
Comment (Mystik01) [reply package.json]: Done ✔️

File: websites/P/POQ/dist/metadata.json [hunk 6]
```diff
@@ -11,21 +11,21 @@
   },
   "service": "POQ",
   "version": "1.0.0",
-  "logo": "https://pbs.twimg.com/profile_images/1303313694546423808/PAaYFaSK_400x400.jpg",
-  "thumbnail": "https://i.gyazo.com/7a15adb09474fd4103ac1a64fa5590cb.jpg",
+  "logo": "https://i.imgur.com/Z3j7Dxl.jpg",
+  "thumbnail": "https://i.imgur.com/rCJpho9.jpg",
```
Comment (Bas950) [inline websites/P/POQ/dist/metadata.json:15]: logo must be 512x512 or bigger

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by ririxidev** [comment]
You literally bumped and add another eslint config to OUR deps so, you need to revert it obviously. Next thing, maybe read about this tho? [else if](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else#description), next thing is you are setting NOTHING to state, this shouldn't happen never. If you will fix your code, you can come back with better pull request.

**Comment by Mystik01** [comment]
Ok sorry

**Pull request closed without merging**